src/beautifiers/prettydiff.coffee

```diff
@@ -109,6 +110,8 @@ module.exports = class PrettyDiff extends Beautifier
           lang = "scss"
         when "TSS"
           lang = "tss"
+        when "Golang Template"
+          lang = "gohtml"
```

Comment:
Regular HTML should work for this: https://github.com/prettydiff/prettydiff/blob/master/lib/language.js#L14